<launch>
    <arg name="input_yaw_rate" default="input_yaw_rate"/>
    <arg name="input_next_cp" default="input_next_cp"/>
    <arg name="input_imu" default="input_imu"/>
    <arg name="input_camera_yaw" default="input_camera_yaw"/>

    <arg name="output_imu" default="output_imu"/>
    <arg name="output_next_cp" default="output_next_cp"/>
    <arg name="output_yaw_rate" default="output_yaw_rate"/>
    <arg name="output_camera_yaw" default="output_camera_yaw"/>


    <node pkg="micro_ros_interface" type="micro_ros_interface_node" name="micro_ros_interface_node"
        output="screen">
        <!-- input  -->
        <remap from="input_yaw_rate" to="$(arg input_yaw_rate)"/>
        <remap from="input_next_cp" to="$(arg input_next_cp)"/>
        <remap from="input_imu" to="$(arg input_imu)"/>

        <!-- output -->
        <remap from="output_yaw_rate" to="$(arg output_yaw_rate)"/>
        <remap from="output_next_cp" to="$(arg output_next_cp)"/>
        <remap from="output_imu" to="$(arg output_imu)"/>
    </node>
</launch>